ecsclient/interface.go

```diff
@@ -0,0 +1,26 @@
+package ecsclient
+
+import "github.com/aws/aws-sdk-go/service/ecs"
+
+// Iface defines the interface required for an object to satisfy in order to perform
+// as the ECSClient struct.
+type Iface interface {
```

Comment:
is this interface ever used?

Reply:
nah, just wondering if this is a nice-to-do